vendor/src/github.com/docker/distribution/registry/client/repository.go

```diff
@@ -604,6 +621,45 @@ func (bs *blobs) Resume(ctx context.Context, id string) (distribution.BlobWriter
 	panic("not implemented")
 }
 
+func (bs *blobs) Mount(ctx context.Context, sourceRepo string, dgst digest.Digest) (distribution.Descriptor, error) {
```

Comment:
Would love to add a comment to Mount() explaining that this is the same as Create() except that the registry checks for the blob across all repositories. No need to update the PR just for this, but would be great to make the change upstream at least.